### DISCUSSION TITLE
Mathjax question.

### DISCUSSION THREAD

**Discussion by QuantLab** [mmistakes/minimal-mistakes#314] [Q&A] [answered]
Does "minimal mistakes" support mathjax? How can it be enabled?

**Comment by mmistakes** [comment, marked as answer]
Not out of the box it doesn't, but it shouldn't be hard for you to add.

According to the [Mathjax documentation](http://docs.mathjax.org/en/latest/start.html) you need to add a JavaScript to the page.

``` html
<script type="text/javascript" async
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-MML-AM_CHTML">
</script>
```

You can place that bit of HTML inside [`/_includes/footer/custom.html`](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/footer/custom.html). It can also go in `/_includes/head/custom.html`, but for page speed reasons I'd put it in the footer unless you come across some sort of conflict.

Hope this helps.

**Reply by TedTschopp** [reply]
I am assuming this can now be done with  `head_scripts` and/or `footer_scripts` as per: https://mmistakes.github.io/minimal-mistakes/docs/configuration/#site-scripts
